### DISCUSSION TITLE
<title>i want to unlink my old bright id & want to add new verifyed id

### DISCUSSION THREAD

**Discussion by shivya1689** [community/community#40697] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

**dear team**, im github user over then 1year, in starting im connected my old unverifyed bright but now i have my verifyed bright id, so want connect new one. according to brightid team only github team are able to help me, so im requesting to you please unlink old my bright id then im able to connect new id, thanks.

**Comment by jfullstackdev** [comment, marked as answer]
what do you mean by verified GitHub ? 

verified email ?

**Reply by shivya1689** [reply]
> what do you mean by verified GitHub ?
> 
> verified email ?

i want only unlink old bright id from this account

**Comment by shivya1689** [comment]
sorry actually im little disturb thats why somthing wrong in post , i want to unlink my old bright id thats it

**Reply by jfullstackdev** [reply]
if that is the email you are referring to, then you go to email settings and remove those emails you want to remove

**Reply by shivya1689** [reply]
no email mate , just i want to unlink my bright id from hithub
